libats-messaging-datatype/src/main/scala/com/advancedtelematic/libats/messaging_datatype/MessageLike.scala

```diff
@@ -0,0 +1,51 @@
+package com.advancedtelematic.libats.messaging_datatype
+
+import io.circe.generic.decoding.DerivedDecoder
+import io.circe.generic.encoding.DerivedObjectEncoder
+import io.circe.parser._
+import io.circe.{Decoder, Encoder}
+import shapeless.Lazy
+
+import scala.reflect.ClassTag
+
+object MessageLike {
+  val partitionPrefixSize = 256
+
+  implicit class StreamNameOp[T <: Class[_]](v: T) {
+    def streamName: String = {
+      v.getSimpleName.filterNot(c => List('$').contains(c))
```

Comment:
Any particular reason for 
```scala
List('$').contains(c)
```
instead of using `==`? 

It doesn't seem to provide any more type safety
```scala
scala> List("hello").contains(true)
res: Boolean = false
```

Reply:
No idea why this was like that